core/app/helpers/spree/base_helper.rb

```diff
@@ -107,6 +107,12 @@ def pretty_time(time)
       [I18n.l(time.to_date, format: :long), time.strftime('%l:%M %p')].join(' ')
     end
 
+    def pretty_date(date)
```

Comment:
@AleksandraKaminska can we just use this method: https://github.com/spree/spree/blob/d30a69879caf87513196a7c6a845a21c4f744cec/backend/app/helpers/spree/admin/base_helper.rb#L161

And move it to `BaseHelper` to make it available in Frontend also?

Reply:
It has different format
<img width="1233" alt="Zrzut ekranu 2020-06-26 o 10 14 27" src="https://user-images.githubusercontent.com/26117538/85836597-1f391200-b797-11ea-8fad-373a325b453b.png">
<img width="1248" alt="Zrzut ekranu 2020-06-26 o 10 15 01" src="https://user-images.githubusercontent.com/26117538/85836606-23fdc600-b797-11ea-84ac-e55aca8a8ffd.png">